### PR TITLE
Added Lubluelu SL60D vacuum device yaml

### PR DESCRIPTION
**Pull request by t34wrj**

### CODE REVIEW

File: custom_components/tuya_local/devices/lublueblu_sl60d_vacuum.yaml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        name: sensor
+        unit: m2
+  - entity: sensor
+    name: Battery
+    class: battery
+    dps:
+      - id: 8
+        type: integer
+        name: sensor
+       unit: "%"
```
Comment (make-all) [inline custom_components/tuya_local/devices/lublueblu_sl60d_vacuum.yaml:140]: This line is not correctly indented.

**Review by make-all — CHANGES_REQUESTED** [review]

**Comment by t34wrj** [comment]
Sorry - I can't see where the misalignment is.

File: custom_components/tuya_local/devices/lublueblu_sl60d_vacuum.yaml (hunk 1 above)
Comment (t34wrj) [reply custom_components/tuya_local/devices/lublueblu_sl60d_vacuum.yaml]: Sorry - have amended the indentation.

**Review by t34wrj — COMMENTED** [review]
Have updated YAML

**Review by make-all — APPROVED** [review]

**Pull request merged**